horreum-backend/src/test/java/io/hyperfoil/tools/horreum/test/PostgresResource.java

```diff
@@ -13,7 +13,7 @@
 public class PostgresResource implements QuarkusTestResourceLifecycleManager {
    private PostgreSQLContainer<?> postgresContainer;
 
-   public static final String POSTGRES_VERSION = "postgres:13";
+   public static final String POSTGRES_VERSION = "postgres:16";
 
```

Comment:
We need remove this constant and resolve the version from the project `pom.xml`: https://github.com/Hyperfoil/Horreum/blob/81c52221edc0530aac21626001d45938a1c67361/pom.xml#L91

For the integration tests, we pass the property in the pom as a system property: https://github.com/Hyperfoil/Horreum/blob/master/horreum-integration-tests/pom.xml#L159

We need something similar for the backend test